third_party/move/move-model/src/ast.rs

```diff
@@ -556,6 +556,12 @@ impl From<Exp> for ExpData {
     }
 }
 
+/// Rewrite result
+pub enum RewriteResult {
+    Rewritten(Exp), // was OK
```

Comment:
Can we remove this 'was' comment? It does not makes sense without knowing the history.

Reply:
Ok.